app/src/main/java/com/nutomic/syncthingandroid/service/RestApi.java

```diff
@@ -124,11 +142,6 @@ public RestApi(Context context, URL url, String apiKey, OnApiAvailableListener a
      */
     private final AtomicInteger mAvailableCount = new AtomicInteger(0);
 
-    /**
-     * Number of asynchronous calls performed in {@link #onWebGuiAvailable()}.
-     */
-    private static final int TOTAL_STARTUP_CALLS = 3;
-
     public interface OnApiAvailableListener {
         void onApiAvailable();
     }
```

Comment:
Is this still used?

Reply:
The interface, is that used?

Reply:
| The interface, is that used? |
Did this overlap with my last post? > Answer is yes, see SynctingService#onApiAvailable